wikimedia/wikipedia-ios
# Diffs - Diff view accessibility update

**Phabricator:** https://phabricator.wikimedia.org/T342359

* This PR is built from https://github.com/wikimedia/wikipedia-ios/pull/4576, so make sure that PR was merged before merging this

### Notes
* This PR adds some VoiceOver adjustments to the Diff view
* We're grouping some labels and adding some context strings to make context clearer
* For now, we're skipping the actual diff content, as the wikitext diff was confusing and sometimes long

### Test Steps
1. Go to a diff screen with VoiceOver on
2. Make sure you can have it read back the contents of the header and the extended header (with user info), and then moves onto the toolbar.
3. Make sure you can access the user button menu

### Screenshots/Videos
